### ISSUE TITLE
Add discussion of report to the documentation

### ISSUE THREAD

**Issue by joeytakeda** [projectEndings/staticSearch#109]
The documentation doesn't describe the report, what it contains, or why you might want to look at it. Once the rest of the diagnostics cleaning up is done (i.e. #27), then we should add some information about that report to the documentation.

**Comment by joeytakeda** [comment]
I've done a bit of this now--we could have more in there, but the div is now in the documentation and referred to by a number of config options.

**Comment by martindholmes** [comment]
The documentation is probably enough for the next release. Do you want to keep this open? If so, shall we push it to a 1.3 milestone?

**Comment by joeytakeda** [comment]
This is probably done for now--if we realize we want more documentation on the report, then we can open a new ticket for improving it. There's still the lingering issue of #27 , so I'll add that we should make sure to update the documentation if/when we split that report into separate files.
